cloud-init.yaml

```diff
@@ -0,0 +1,9 @@
+#cloud-config
```

Comment:
You probably don't want to check this in since it get's created and delete by the create/destroy vm. Add the filename cloud-init.yaml to the .gitignore and do a `git rm --cached cloud-init.yaml` on your next PR.